scipp-imaging-2D.ipynb

```diff
@@ -8,186 +8,329 @@
     "\n",
     "Before starting you must:\n",
     "- Ensure that `scipp` and `mantid` are on your `PYTHONPATH`.\n",
-    "- Generate the `config.py` file using `make_config.py`. Refer to the `README.md` or `python make_config.py --help` for information.\n",
-    "- Install dependency `fabio`: `conda install fabio` (used for image loading)\n",
+    "- Generate the `scippconfig.py` file using `make_config.py` located in same directory as this notebook. Refer to the `README.md` or `python make_config.py --help` for information.\n",
+    "- Install dependencies : `conda install fabio tifffile` (used for image handling)\n",
     "\n",
     "For `scipp` and `mantid` follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
     "\n",
-    "Converted to use scipp and notebook from [this file](https://git.esss.dk/testbeamline/gp2/blob/1c69213b1124982bbbe762da9c6c6457a49f2a92/reduce.py) by Dimitar Tasev on 2020-01-13"
+    "Converted to use scipp and notebook from [this repository](https://git.esss.dk/testbeamline/gp2/-/tree/stressexperiment).\n",
+    "\n",
+    "For Table of Contents install Jupyter extensions then reload the notebook:\n",
+    "`conda install -c conda-forge jupyter_contrib_nbextensions`\n",
+    "`jupyter contrib nbextension install --user`\n",
+    "`jupyter nbextension enable toc2/main`\n",
+    "\n",
+    "# Experimental Summary\n",
```

Comment:
This section/cell appears twice?

Reply:
Oh, thanks for noticing, probably messed up a merge somewhere.